mlflow/models/python_api.py

```diff
@@ -75,6 +76,9 @@ def build_docker(
             The version of installed mlflow will be the same as the one used to invoke this command.
 
         mlflow_home: Path to local clone of MLflow project. Use for development only.
+
+        base_image: Base image for the Docker image. If not specified, the default image is either
```

Comment:
Let's add a note that non-ubuntu base image might not work :)

Reply:
done! thanks